alacritty_terminal/src/tty/windows/mod.rs

```diff
@@ -266,15 +308,15 @@ impl EventedReadWrite for Pty {
 
     #[inline]
     fn deregister(&mut self, poll: &mio::Poll) -> io::Result<()> {
-        poll.deregister(&self.conout)?;
-        poll.deregister(&self.conin)?;
+        poll.deregister(self.conout())?;
+        poll.deregister(self.conin())?;
         poll.deregister(self.child_watcher.event_rx())?;
         Ok(())
     }
 
     #[inline]
     fn reader(&mut self) -> &mut Self::Reader {
-        &mut self.conout
+        self.conout_mut()
```

Comment:
Why do we have a `conout_mut` function here, when it is never used anywhere else? Why wouldn't we inline it? Is it solely because we have a `conout` method that is used in other places, making the API more consistent internally? Same goes for conin.